independent-publisher-connectors/U.K. Government Bank Holidays/UK-Bank-Holidays.swagger.json

```diff
@@ -1,55 +1,97 @@
-{
-  "swagger": "2.0",
-  "info": {
-    "title": "UK Bank Holidays",
-    "description": "A JSON object containing the bank holidays of the four kingdoms of the United Kingdom supplied by H.M. Government",
-    "version": "1.0",
-    "contact": {
-      "name": "Martyn Lesbirel",
-      "email": "ipc-author-bh@outlook.com"
-    }
-  },
-  "host": "www.gov.uk",
-  "schemes": [
-    "https"
-  ],
-  "consumes": [],
-  "produces": [],
-  "paths": {
-    "/bank-holidays.json": {
-      "get": {
-        "responses": {
-          "default": {
-            "description": "default",
-            "schema": {}
-          }
-        },
-        "summary": "All holidays for all kingdoms",
-        "description": "All bank holidays for all kingdoms as defined the Government",
-        "operationId": "AllKingdomHolidays",
-        "x-ms-visibility": "important",
-        "parameters": []
-      }
-    }
-  },
-  "x-ms-connector-metadata": [
-    {
-      "propertyName": "WebSite",
-      "propertyValue": "https://www.gov.uk/bank-holidays"
-    },
-    {
-      "propertyName": "Privacy policy",
-      "propertyValue": "https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/"
-    },
-    {
-      "propertyName": "Categories",
-      "propertyValue": "Data;Productivity"
-    }
-  ],
-  "definitions": {},
-  "parameters": {},
-  "responses": {},
-  "securityDefinitions": {},
-  "security": [],
-  "tags": []
-}
\ No newline at end of file
+swagger: '2.0'
```

Comment:
@troystaylor Your connector swagger is not in a valid JSON format. Please revert it back to the previous version.

Reply:
Not sure that happened, but fixed